src/benchmarks/micro/libraries/System.Net.Sockets/Perf.Socket.cs

```diff
@@ -116,6 +117,43 @@ public async Task ReceiveAsyncThenSendAsync_SocketAsyncEventArgs()
             }
         }
 
+        [Benchmark]
+        public async Task ReceiveFromAsyncThenSendToAsync_Task()
+        {
+            ReadOnlyMemory<byte> clientBuffer = new byte[1];
+            Memory<byte> serverBuffer = new byte[1];
+
+            EndPoint ep = new IPEndPoint(IPAddress.None, 0);
```

Comment:
These are using the older EndPoint APIs. What is your intent regarding the new APIs you're adding?